CRM/Core/Payment/Realex.php

```diff
@@ -185,14 +184,13 @@ public function doPayment(&$params, $component = 'contribute') {
       'trxn_result_code' => $response['RESULT'],
     ];
 
-    $params['trxn_id'] = $response['PASREF'];
     $params['trxn_result_code'] = serialize($extras);
     $params['currencyID'] = $this->_getParam('currency');
-    $params['fee_amount'] = 0;
```

Comment:
Also currencyID here same as trxn_result_code.

Reply:
See comment above - have now added link to lab issue where we agreed return parameters.